types/react-native-razorpay/index.d.ts

```diff
@@ -0,0 +1,92 @@
+// Type definitions for react-native-razorpay 2.2
+// Project: https://www.npmjs.com/package/react-native-razorpay
+// Definitions by: Ankan Bhattacharya <https://github.com/Ankan002>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.4
```

Comment:
please use intermediary type to avoid TS version restrictions impacting usage:
```ts
type Notes = Record<string | number, string | number>;
```
and:
```ts
notes?: Notes;
```

Reply:
@peterblazejewicz Thanks for reviewing and suggesting the required changes. I have implemented those changes, once you get time please review the changes.